lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
+
+  module.exports = {
+    getRange: function(ed) {
+      var buffer, cursor, end, regex, regexString, start;
+      buffer = ed.getBuffer();
+      start = buffer.getFirstPosition();
+      end = buffer.getEndPosition();
+      regexString = atom.config.get('julia-client.cellDelimiter');
+      regex = new RegExp(regexString);
+      cursor = ed.getCursorBufferPosition();
+      if (cursor.row > 0) {
+        buffer.backwardsScanInRange(regex, [start, cursor], function(arg) {
+          var range;
+          range = arg.range;
+          return start = range.start;
+        });
+      }
+      buffer.scanInRange(regex, [cursor, end], function(arg) {
+        var range;
+        range = arg.range;
+        return end = range.start;
```

Comment:
This will attach the result to the next line with a cell delimiter in it, which is fine, but currently block results are displayed *under* that line, which I find confusing. It'd be trivial to add a patch to ink to allow displaying block results above the marker though. 
If you don't want to look into that codebase I can push a commit for that.